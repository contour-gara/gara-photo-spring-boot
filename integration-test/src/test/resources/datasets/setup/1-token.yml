token:
  - id: 1
    access_token: access-token
    refresh_token: refresh-token
    date_time: "2024-04-14T19:48:34.000+09:00[Asia/Tokyo]"
