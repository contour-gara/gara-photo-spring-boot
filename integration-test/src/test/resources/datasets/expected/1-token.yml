token:
  - id: 1
    access_token: access-token
    refresh_token: refresh-token
    date_time: "2024-04-23T00:00:00+09:00[Asia/Tokyo]"
